platform/darwin/src/MGLMetricsManager.m

```diff
@@ -0,0 +1,57 @@
+#import "MGLMetricsManager_Private.h"
+#import "MGLNetworkConfiguration_Private.h"
+#if TARGET_OS_IOS
+#import "MGLMapboxEvents.h"
+#endif
+
+NSString* MGLStringFromMetricType(MGLMetricType metricType) {
+    return [@[kMGLDownloadPerformanceEvent] objectAtIndex:metricType];
```

Comment:
In this case we could just return `kMGLDownloadPerformanceEvent` and check that `metricType == 0`. We could return `nil` if not.

Ideally, we should check to ensure that the the size of the array and the count of `MGLMetricType`s are the same.